src/components/FilterDrawer/FilterDrawer.js

```diff
@@ -15,14 +16,26 @@ import { connect } from 'react-redux';
 import { PLURAL_RESOURCE_TYPES } from '../../resources/resourceTypes';
 import Colors from '../../constants/Colors';
 import { toggleResourceTypeFilter } from '../../redux/epics';
+import { orderedResourceTypeFiltersSelector } from '../../redux/selectors';
 
 const ResourceTypeFilter = ({ resourceType, filterOpen, toggleResourceTypeFilterAction }) => {
   const label = PLURAL_RESOURCE_TYPES[resourceType];
+  let thumbColor;
+  if (filterOpen) {
+    thumbColor = Platform.OS === 'ios' ? 'white' : Colors.primary;
+  } else {
+    thumbColor = Platform.OS === 'ios' ? 'white' : Colors.lightgrey;
+  }
```

Comment:
iOS is always white?
is it only android that gets different colors?
(should  we test for Android, rather than !iOS?)

Reply:
Yes, if you look at iOS, the thumbButton staying white is typical. Also it looks nice. For android, the button needs a color because it disappears in the background.